roles/launch_velero/tasks/main.yml

```diff
@@ -79,4 +79,6 @@
   k8s:
     state: "{{ state }}"
     definition: "{{ lookup('file', '{{ playbook_dir }}/velero/config/minio/30-restic-daemonset.yaml')}}"
+- name: Install ocpmigrate plugin
+  shell: "{{ playbook_dir }}/velero/velero plugin add quay.io/ocpmigrate/velero-plugin"
 
```

Comment:
This needs a conditional, or the destroy-velero.yml playbook will fail:
`when: state == "present"`